CardinalKit-Example/CardinalKit/Library/CareKit/CKCareKitRemoteSyncWithFirestore.swift

```diff
@@ -10,6 +10,8 @@ import Foundation
 import CareKit
 import CareKitStore
 import CareKitUI
+import CardinalKit
+import FirebaseFirestore
 
 class CKCareKitRemoteSyncWithFirestore: OCKRemoteSynchronizable {
```

Comment:
I have not tested these changes yet, 
but wanted to note that the OCKRemoteSynchronizable protocol was introduced here:
https://developer.apple.com/videos/play/wwdc2020/10151/

Have we encountered any challenges with the protocol?